spec/features/idv/doc_auth/hybrid_handoff_spec.rb

```diff
@@ -536,7 +522,6 @@ def verify_no_upload_photos_section_and_link(page)
         context 'when sp ipp is not available' do
           let(:sp_ipp_enabled) { false }
           context 'when selfie is disabled system wide' do
```

Comment:
@n1zyy Actually I think this is another case to remove?

Reply:
It looks like we have 2 contexts testing `biometric_comparison_required` in here with this change. Could we reduce this down to a context for `biometric_comparison_required=true` and `biometric_comparison_required=false`? Same for the other places where this pattern appears in these files.

> sounds like it references the feature flag, but the test does no such thing

I believe that was done in the before action here: https://github.com/18F/identity-idp/pull/10751/files#diff-a1e3ac07193f752ccd3c694642c27b35d267edc95fdb7e260216b248951e6fbaL343